src/button/README.md

```diff
@@ -2,7 +2,48 @@
+| --------------------------------------------------------------- | ------------------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png)          | ![Image placeholder](/docs/assets/img-placeholder.png)             |
+| Don't use multiple primary buttons unless absolutely necessary. | Provide a single primary button with one or more secondary actions |
+
+| ❌ Don't                                                            | ✅ Do                                                                                                                                             |
+| ------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png)              | ![Image placeholder](/docs/assets/img-placeholder.png)                                                                                            |
+| Don't override button widths to always fill their parent container. | Allow buttons to fill a [VS Code View](https://code.visualstudio.com/api/references/extension-guidelines#views) until a maximum width is reached. |
+
+| ❌ Don't                                               | ✅ Do                                                  |
+| ------------------------------------------------------ | ------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png) |
+| Don't use all caps or lowercase text.                  | Use sentence case for all button text.                 |
+
+| ❌ Don't                                               | ✅ Do                                                                                          |
+| ------------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png)                                         |
+| Don't use vague action text.                           | Use clear verbs like "Save" or "Cancel" to ensure users feel confident when peforming actions. |
+
+| ❌ Don't                                               | ✅ Do                                                        |
+| ------------------------------------------------------ | ------------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png) | ![Image placeholder](/docs/assets/img-placeholder.png)       |
+| Don't use buttons as navigational elements.            | Use buttons to perform actions relevant to the current view. |
```

Comment:
Do we want to leave a link saying which component should be used for navigation (I assume `vscode-link`)? Just like you did with the badge and tag components.

Reply:
Good call 👍